<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Target IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="connectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="statusBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="toggleGreenLEDBtn">
      <property name="text">
       <string>Toggle green LED</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toggleRedLEDBtn">
      <property name="text">
       <string>Toggle red LED</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readGreenLEDBtn">
      <property name="text">
       <string>Read green LED</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
